### PR TITLE
[11.0][MIG] dead_mans_switch_client

### PR DESCRIPTION
**Pull request by aisopuro**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis and runbot status

**Comment by aisopuro** [comment]
Sorry @pedrobaeza I started with just the patch-commits, so those were destined to fail. I've now migrated the code, so Travis is giving the green light, but Runbot seems to have failed. I was able to open and log into the Runbot, and it seemed to work fine. I wasn't able to download any log files. What can I do to fix the runbot error?

**Comment by pedrobaeza** [comment]
You can check here the logs:

https://runbot2.odoo-community.org/runbot/static/build/3327863-1277-e764df/logs/job_20_test_all.txt

It's a warning as ir.todo has changed its structure:

```
2018-06-05 11:15:39,848 166 WARNING openerp_test odoo.models: ir.actions.todo.create() includes unknown fields: type
```

**Review by mlaitinen — APPROVED** [review]
Nice :+1:

File: dead_mans_switch_client/data/ir_actions.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <act_window id="action_setup" res_model="ir.config_parameter"
+                name="Configure the dead man's switch server" view_mode="form"
+                context="{'default_key': 'dead_mans_switch_client.url'}"/>
+    <record id="todo_setup" model="ir.actions.todo">
+        <field name="name">Configure the dead man's switch server</field>
+        <field name="type">automatic</field>
```
Comment (mlaitinen) [inline dead_mans_switch_client/data/ir_actions.xml]: This field doesn't exist in `11.0`. Did you test that this `ir.todo` launches after update?

**Review by mlaitinen — CHANGES_REQUESTED** [review]
Small detail. Need to have 0 warnings during build.

**Review by mlaitinen — APPROVED** [review]

**Comment by aisopuro** [comment]
This still seems to be hanging around. Is there anyone who would be willing to be a second reviewer? Does @pedrobaeza know?

**Comment by pedrobaeza** [comment]
OK, let's go for an optimistic merging.

**Pull request merged**